Version 1.7.0 fails to build

**Description**
I was trying to upgrade to version 1.7.0 from version 1.6.0 and when building newtmgr the build fails whereas building 1.6.0 works correctly.  I believe this has to do with the fact that version 1.7.0 has moved to using go modules.
```
➜  mynewt-newtmgr git:(a222bac) git status
HEAD detached at mynewt_1_7_0_tag
nothing to commit, working tree clean
➜  mynewt-newtmgr git:(a222bac) make
mkdir -p /tmp/mynewt.7Tu55dJ2caG/src/mynewt.apache.org
ln -s /home/bjohnson/Development/git/mynewt/mynewt-newtmgr /tmp/mynewt.7Tu55dJ2caG/src/mynewt.apache.org/newtmgr
GOPATH=/tmp/mynewt.7Tu55dJ2caG go get github.com/currantlabs/ble github.com/mgutz/logxi/v1 golang.org/x/sys/unix
go: finding github.com/mattn/go-runewidth v0.0.2
go: finding github.com/mitchellh/go-homedir v0.0.0-20161203194507-b8bc1bf76747
go: finding github.com/sirupsen/logrus v1.4.1
go: finding github.com/fatih/color v1.5.0
go: finding github.com/abiosoft/readline v0.0.0-20180607040430-155bce2042db
go: finding github.com/flynn-archive/go-shlex v0.0.0-20150515145356-3f9db97f8568
go: finding github.com/fatih/structs v1.0.0
go: finding github.com/pkg/errors v0.8.0
go: finding gopkg.in/cheggaaa/pb.v1 v1.0.28
go: finding github.com/runtimeco/go-coap v0.0.0-20180109101419-47c23e2a63ea
go: finding mynewt.apache.org/newt v0.0.0-20190515182908-60fc2df06bfc
go: finding github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
go: finding github.com/joaojeronimo/go-crc16 v0.0.0-20140729130949-59bd0194935e
go: finding github.com/raff/goble v0.0.0-20170814221208-b12b34f940c4
go: finding github.com/spf13/cobra v0.0.3
go: finding github.com/ugorji/go v0.0.0-20170826155943-8c0409fcbb70
go: finding github.com/konsorten/go-windows-terminal-sequences v1.0.1
go: finding golang.org/x/sys v0.0.0-20180905080454-ebe1bf3edb33
go: finding gopkg.in/abiosoft/ishell.v1 v1.0.0-20171224170712-50251d04cb42
go: finding github.com/stretchr/testify v1.2.2
go: finding github.com/tarm/serial v0.0.0-20170530023055-e50d7d20b1f6
go: finding github.com/mattn/go-isatty v0.0.2
go: finding github.com/abiosoft/ishell v2.0.0+incompatible
go: finding github.com/chzyer/logex v1.1.10
go: finding github.com/pmezard/go-difflib v1.0.0
go: finding github.com/go-ble/ble v0.0.0-20181002102605-e78417b510a3
go: finding github.com/davecgh/go-spew v1.1.1
go: finding github.com/sirupsen/logrus v1.0.5
go: finding github.com/stretchr/objx v0.1.1
go: finding golang.org/x/net v0.0.0-20170828231752-66aacef3dd8a
go: finding github.com/mgutz/logxi v0.0.0-20161027140823-aebf8a7d67ab
go: finding github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1
go: finding github.com/spf13/pflag v1.0.3
go: finding golang.org/x/crypto v0.0.0-20180621125126-a49355c7e3f8
go: finding github.com/chzyer/readline v0.0.0-20171208011716-f6d7a1f6fbf3
go: finding github.com/spf13/pflag v1.0.1
go: finding golang.org/x/sys v0.0.0-20180707002001-3c6ecd8f22c6
go: finding github.com/mattn/go-colorable v0.0.9
go: finding github.com/cheggaaa/pb v1.0.18
go: finding github.com/spf13/cast v1.2.0
go: finding github.com/NickBall/go-aes-key-wrap v0.0.0-20170929221519-1c3aa3e4dfc5
go: finding github.com/inconshreveable/mousetrap v1.0.0
go: finding github.com/mgutz/logxi latest
go: downloading github.com/mgutz/logxi v0.0.0-20161027140823-aebf8a7d67ab
go: finding golang.org/x/sys/unix latest
go: finding golang.org/x/sys latest
go: downloading golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
go: extracting github.com/mgutz/logxi v0.0.0-20161027140823-aebf8a7d67ab
go: finding github.com/currantlabs/ble latest
go: extracting golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
go: downloading github.com/currantlabs/ble v0.0.0-20171229162446-c1d21c164cf8
go: downloading github.com/mattn/go-colorable v0.0.9
go: downloading github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
go: downloading github.com/mattn/go-isatty v0.0.2
go: extracting github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
go: extracting github.com/mattn/go-colorable v0.0.9
go: extracting github.com/mattn/go-isatty v0.0.2
go: extracting github.com/currantlabs/ble v0.0.0-20171229162446-c1d21c164cf8
go: downloading github.com/pkg/errors v0.8.0
go: downloading golang.org/x/net v0.0.0-20170828231752-66aacef3dd8a
go: extracting github.com/pkg/errors v0.8.0
go: extracting golang.org/x/net v0.0.0-20170828231752-66aacef3dd8a
cd /tmp/mynewt.7Tu55dJ2caG/src/mynewt.apache.org/newtmgr/newtmgr && GOPATH=/tmp/mynewt.7Tu55dJ2caG GO15VENDOREXPERIMENT=1 go install
cli/image.go:29:2: cannot find package "github.com/cheggaaa/pb" in any of:
	/usr/lib/go-1.12/src/github.com/cheggaaa/pb (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/cheggaaa/pb (from $GOPATH)
../nmxact/omp/omp.go:27:2: cannot find package "github.com/fatih/structs" in any of:
	/usr/lib/go-1.12/src/github.com/fatih/structs (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/fatih/structs (from $GOPATH)
bll/bll_common.go:29:2: cannot find package "github.com/go-ble/ble" in any of:
	/usr/lib/go-1.12/src/github.com/go-ble/ble (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/go-ble/ble (from $GOPATH)
bll/bll_xport.go:27:2: cannot find package "github.com/go-ble/ble/examples/lib/dev" in any of:
	/usr/lib/go-1.12/src/github.com/go-ble/ble/examples/lib/dev (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/go-ble/ble/examples/lib/dev (from $GOPATH)
bll/bll_xport_linux.go:26:2: cannot find package "github.com/go-ble/ble/linux" in any of:
	/usr/lib/go-1.12/src/github.com/go-ble/ble/linux (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/go-ble/ble/linux (from $GOPATH)
bll/bll_xport_linux.go:27:2: cannot find package "github.com/go-ble/ble/linux/hci/cmd" in any of:
	/usr/lib/go-1.12/src/github.com/go-ble/ble/linux/hci/cmd (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/go-ble/ble/linux/hci/cmd (from $GOPATH)
../nmxact/mtech_lora/mtech_lora_sesn.go:31:2: cannot find package "github.com/joaojeronimo/go-crc16" in any of:
	/usr/lib/go-1.12/src/github.com/joaojeronimo/go-crc16 (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/joaojeronimo/go-crc16 (from $GOPATH)
config/connprofile.go:24:2: cannot find package "github.com/mitchellh/go-homedir" in any of:
	/usr/lib/go-1.12/src/github.com/mitchellh/go-homedir (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/mitchellh/go-homedir (from $GOPATH)
nmutil/nmutil.go:25:2: cannot find package "github.com/pkg/errors" in any of:
	/usr/lib/go-1.12/src/github.com/pkg/errors (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/pkg/errors (from $GOPATH)
cli/interactive.go:25:2: cannot find package "github.com/runtimeco/go-coap" in any of:
	/usr/lib/go-1.12/src/github.com/runtimeco/go-coap (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/runtimeco/go-coap (from $GOPATH)
cli/commands.go:25:2: cannot find package "github.com/sirupsen/logrus" in any of:
	/usr/lib/go-1.12/src/github.com/sirupsen/logrus (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/sirupsen/logrus (from $GOPATH)
cli/commands.go:26:2: cannot find package "github.com/spf13/cobra" in any of:
	/usr/lib/go-1.12/src/github.com/spf13/cobra (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/spf13/cobra (from $GOPATH)
../nmxact/nmserial/serial_xport.go:35:2: cannot find package "github.com/tarm/serial" in any of:
	/usr/lib/go-1.12/src/github.com/tarm/serial (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/tarm/serial (from $GOPATH)
../nmxact/nmxutil/nmxutil.go:34:2: cannot find package "github.com/ugorji/go/codec" in any of:
	/usr/lib/go-1.12/src/github.com/ugorji/go/codec (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/github.com/ugorji/go/codec (from $GOPATH)
bll/bll_sesn.go:31:2: cannot find package "golang.org/x/net/context" in any of:
	/usr/lib/go-1.12/src/golang.org/x/net/context (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/golang.org/x/net/context (from $GOPATH)
cli/interactive.go:27:2: cannot find package "gopkg.in/abiosoft/ishell.v1" in any of:
	/usr/lib/go-1.12/src/gopkg.in/abiosoft/ishell.v1 (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/gopkg.in/abiosoft/ishell.v1 (from $GOPATH)
newtmgr.go:28:2: cannot find package "mynewt.apache.org/newt/util" in any of:
	/usr/lib/go-1.12/src/mynewt.apache.org/newt/util (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/mynewt.apache.org/newt/util (from $GOPATH)
../nmxact/nmble/ble_xport.go:30:2: cannot find package "mynewt.apache.org/newt/util/unixchild" in any of:
	/usr/lib/go-1.12/src/mynewt.apache.org/newt/util/unixchild (from $GOROOT)
	/tmp/mynewt.7Tu55dJ2caG/src/mynewt.apache.org/newt/util/unixchild (from $GOPATH)
Makefile:28: recipe for target 'build' failed
make: *** [build] Error 1
```

**Operating system**
Linux Mint 19.1

**Go Version**
go version go1.12.8 linux/amd64

--- a/Makefile
+++ b/Makefile
@@ -28,8 +28,9 @@ build:
 	mkdir -p $(MYNEWTDIR)
 	ln -s $(NEWTMGR_INSTALLDIR) $(REPODIR)
 	GOPATH=$(GODIR) go get github.com/currantlabs/ble github.com/mgutz/logxi/v1 golang.org/x/sys/unix
-	cd $(NEWTMGRDIR) && GOPATH=$(GODIR) GO15VENDOREXPERIMENT=1 go install
+	cd $(NEWTMGRDIR) && GOPATH=$(GODIR) GO15VENDOREXPERIMENT=1 GO111MODULE=on go install
 	mv $(GODIR)/bin/$(TARGET) $(DSTFILE)
+	chmod -R 777 $(GODIR)
 	rm -rf $(GODIR)
 
 install:

